include/CppUTest/StandardCLibrary.h

```diff
@@ -1,8 +1,10 @@
 
+/* Must include this first to ensure the StandardC include in CppUTestConfig still happens at the right moment */
+#include "CppUTestConfig.h"
```

Comment:
It's fine to not include TestHarness, but it should be including CppUTestConfig instead of StandardCLibrary.  The latter is an implementation detail of the config header, and shouldn't be used directly.  At least that's the structure as it appears to me.